### PR TITLE
[CHORE] Update to react-native 0.60.4

### PR DESCRIPTION
**Pull request by diegolmello**
@RocketChat/ReactNative

Update RN and other libs.

### CODE REVIEW

File: android/settings.gradle [hunk 1]
```diff
@@ -1,45 +1,12 @@
 apply from: '../node_modules/react-native-unimodules/gradle.groovy'
 includeUnimodulesProjects()
 
-rootProject.name = 'RocketChatRN'
-include ':react-native-document-picker'
-project(':react-native-document-picker').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-document-picker/android')
+rootProject.name = 'EmptyRN'
```
Comment (djorkaeffalexandre) [inline android/settings.gradle]: why?

File: android/app/src/main/java/chat/rocket/reactnative/MainActivity.java [hunk 2]
```diff
@@ -11,11 +11,11 @@
...
 public class MainActivity extends ReactFragmentActivity {
 
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        SplashScreen.show(this);
-        super.onCreate(null);
-    }
+    // @Override
+    // protected void onCreate(Bundle savedInstanceState) {
+    //     SplashScreen.show(this);
```
Comment (djorkaeffalexandre) [inline android/app/src/main/java/chat/rocket/reactnative/MainActivity.java]: this works?

**Comment by lorek123** [comment]
Did you check the Hermes JS engine? https://facebook.github.io/react-native/docs/hermes
Maybe it's a right moment to check it.

**Comment by diegolmello** [comment]
> Did you check the Hermes JS engine? https://facebook.github.io/react-native/docs/hermes
> Maybe it's a right moment to check it.

Hermes won't work with Realm right now.

**Pull request merged**